src/python/WMCore/Credential/Proxy.py

```diff
@@ -682,7 +685,7 @@ def vomsExtensionRenewal(self, proxy, voAttribute='cms'):
 
         vomsValid = '00:00'
         if vomsTime > 0:
-            vomsValid = "%d:%02d" % (vomsTime / 3600, (vomsTime - (vomsTime / 3600) * 3600) / 60)
+            vomsValid = "%d:%02d" % (int(vomsTime // 3600), int((vomsTime % 3600) // 60))
```

Comment:
For the record, am I wrong or you actually made a bug fix here?

Reply:
doesn't python have any standard lib to convert seconds to hours:min format ? Learning proper use of // vs old_div vs % is surely good, but should not be needed here.

Reply:
We could use `time` or `datetime`, but I can not find a way to use `strftime` to print the hours without overloading into the day field if `%H > 23`. For example

```python
# This is ok
>>> import time
>>> time.strftime('%H:%M', time.gmtime(10000))
'02:46'
>>> str(datetime.timedelta(seconds=10000))
'2:46:40'

# this is less ok
>>> time.strftime('%H:%M', time.gmtime(100000))
'03:46'
>>> str( datetime.timedelta(seconds=100000) )
'1 day, 3:46:40'
```

If our proxies expire in less than 24 hours than this is a valid option, even though it feels like a bit binding.

Did you have something else in mind? Am I missing a `strftime` parameter?